docs/development.md

```diff
@@ -0,0 +1,109 @@
+# Development Guide
+
+This document is a guide to start developing the Kvrocks operator.
+
+## Set up the development environment
+
+We recommend you use a Linux/macOS platform for development.
+
+### Install Required Software
+-   Go
+    
+    -   Currently, building the Kvrocks operator requires Go 1.17 or later.
+-   Docker
+-   Kubernetes cluster
+    
+    -   You can use the minikube to provision your local Kubernetes cluster
+
+## Start debugging
+
+Before getting started, please run the following commands to perform some checks:
+
+### Check the Kubernetes cluster
+
+```bash
+kubectl version --short
+```
+
+### Install the local manifests
+
+The local manifests contain the CRD, run the following command to install it:
+
+```bash
+make install
+```
+
+Expected output:
+```bash
+kvrocks.kvrocks.com                        2023-04-22T06:23:33Z
```

Comment:
Can change the `kvrocks.kvrocks.com` to `kvrocks.apache.org`

Reply:
Sure, but this name is a name defined by [CRD](https://github.com/RocksLabs/kvrocks-operator/blob/unstable/deploy/crd/templates/crd.yaml#L8), so I think it would be more appropriate to submit a new PR to modify this name.

Reply:
Yes, it makes sense.